tgui/packages/tgui/interfaces/GasTank.js

```diff
@@ -1,8 +1,7 @@
-import { Fragment } from 'inferno';
 import { useBackend } from '../backend';
-import { Box, Section, AnimatedNumber } from '../components';
+import { Section, RoundGauge, LabeledList } from '../components';
```

Comment:
```suggestion
import { LabeledList, RoundGauge, Section } from '../components';
```
Pedantic? Yes. Alphabetizing was part of one of the tgui cleanup PR ports, though, so trying to keep us consistent there.